curandommatrices/src/circular.jl

```diff
@@ -0,0 +1,61 @@
+# FIXME: this can be accelerated
```

Comment:
How? Some insights?

Reply:
No insight, only based on the following warning
```
┌ Warning: Performing scalar operations on GPU arrays: This is very slow, consider disallowing these operations with `allowscalar(false)`
└ @ GPUArrays ~/.julia/packages/GPUArrays/J4c3Q/src/indexing.jl:16
```